include/common/internal/transport/serialization_transport.h

```diff
@@ -62,7 +62,10 @@ typedef enum
 {
     SERIALIZATION_COMMAND = 0,
     SERIALIZATION_RESPONSE = 1,
-    SERIALIZATION_EVENT = 2
+    SERIALIZATION_EVENT = 2,
+    SERIALIZATION_DTM_CMD = 3,
```

Comment:
These names are a bit cryptic to me. Perhaps we can add comments to each of these?

Reply:
Not sure what to add, except that DTM = direct test mode.